pkg/clientaccess/token.go

```diff
@@ -136,12 +139,16 @@ func parseToken(token string) (*Info, error) {
 	parts := strings.SplitN(token, "::", 2)
 	token = parts[0]
 	if len(parts) > 1 {
+		hashLen := len(parts[0])
+		if hashLen > 0 && hashLen != caHashLength {
+			return nil, fmt.Errorf("invalid token CA hash length")
```

Comment:
Can you include the erroneous length in the message? Or, change the `fmt.Errorf` to `errors.New`.